code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -172,6 +172,7 @@
 	var/has_glowing_eyes = 0								// Whether the eyes are shown above all lighting
 	var/water_movement = 0									// How much faster or slower the species is in water
 	var/snow_movement = 0									// How much faster or slower the species is on snow
+	var/waterbreathing = 0									// Vorestation edit. Allows for what you think.
```

Comment:
Above.

Reply:
I am not sure what you are requesting by this. The var needs to be defined somewhere, no?